packages/server/src/tests/epics/epicSelection.test.ts

```diff
@@ -869,22 +866,22 @@ describe('bandit null hypothesis', () => {
 
     const abTestTest: EpicTest = {
         ...testDefault,
-        name: AB_TEST_TEST_NAME,
+        name: NonStickyVariantsTestNames.Sticky,
         articlesViewedSettings: undefined,
         variants: variants,
     };
 
     const banditTest: EpicTest = {
         ...testDefault,
-        name: BANDIT_TEST_NAME,
+        name: NonStickyVariantsTestNames.NonSticky,
         isBanditTest: true,
         articlesViewedSettings: undefined,
         variants: variants,
     };
 
     const tests = [abTestTest, banditTest];
 
-    it('should return AB test and bandit ~ equally', () => {
+    it('should return sticky and non-sticky ~ equally', () => {
```

Comment:
I've only kept one of the old bandit tests here - it checks that browserIds are evenly spread across the two "tests".
I'm not sure it's worth updating the other tests, so I removed them